### ISSUE TITLE
Having issue with the nvm use 11.0.0 command

### ISSUE THREAD

**Issue by Mexxy-lab** [insidesherpa/JPMC-tech-task-2#12]
The command keeps giving me an this message (**e**xec: "C:\\nodejs\\nvm\\elevate.cmd": file does not exist).

How do i go about it? Please note i am using python v3.7.3 and i was able to install the nvm which is currently showing its running version 1.1.7.

After installing v11.0.0 successfully, on trying to use it that's when i get this message. Any help would be greatly appreciated. See screenshot below.

![image](https://user-images.githubusercontent.com/49109013/69389589-7a9fa080-0c9a-11ea-870f-34c084d0438b.png)

**Comment by jbf-insidesherpa** [comment]
@Mexxy-lab your issue is with the path where nvm is installed. It's installed in the user "FECHI PC". nvm in Windows has issues with directories having a space in their name. Perhaps you reinstall nvm and put your nvm in a public account in your computer that has no space in the name

**Comment by Mexxy-lab** [comment]
I have done this before but wasn't successful with the fix. See screenshot i got below after executing the nvm use 11.0.0 command after successful installation:

![image](https://user-images.githubusercontent.com/49109013/69426752-f7af3200-0cfb-11ea-9538-8fd38a684d90.png)
![image](https://user-images.githubusercontent.com/49109013/69426792-11507980-0cfc-11ea-8a07-c3f31e5bb88d.png)

**Comment by jbf-insidesherpa** [comment]
@Mexxy-lab looking at your issue it seems there have been related discussions about it in the windows nvm repo:

- https://github.com/coreybutler/nvm-windows/issues/198
- https://github.com/nvm-sh/nvm/issues/1896

Unfortunately I don't think these threads provide a definitive solution (yet).

My suggestion for you to be able to work around this problem is to download the (windows) node_modules task 2 [here](https://drive.google.com/open?id=1ML6nECK18J0jM3hhfJdSCRtCNLssruYQ) and just place it in your copy of the repository as node_modules (if your repo already has, then delete the existing one and replace it with the one you downloaded).

You can then just skip doing **npm install** and jump straight to **npm start**

**Comment by Mexxy-lab** [comment]
I downloaded, extracted the files and stored it in the repository as node_modules as mentioned but still the npm command isn't running/starting it keeps giving this message npm isn't recognized as an internal or external command operable program or batch file. Is there something i am doing wrong still.

PS: The python version i have installed is version 3.7 64bit, do i need to switch to the latest version? Could this be what is affecting me. Thanks for your help so far.

![image](https://user-images.githubusercontent.com/49109013/69471060-68406800-0d69-11ea-9f9d-9a5134dfe545.png)

**Comment by Mexxy-lab** [comment]
Hello Team,

I fixed it finally and hoping this would be the final fix. I uninstalled the initial NVM which was installed previously and re-installed it this time specifying the new path and Symlink path to folders without white spaces in the folder path name.

After the installation, i opened cmd as an administrator and performed the nvm installation of v11.0.0 and nvm use v11.0.0 and it fixed it. See below image:

![image](https://user-images.githubusercontent.com/49109013/69471734-b86ef880-0d70-11ea-942c-9d9d6de38bf0.png)

Thanks once more for your help.
